src/app/contribute/contribution-process/index.html

```diff
@@ -90,6 +96,12 @@
         <p>
           Create <a stacheRouterLink="/contribute/contribution-process/guidelines" fragment="demos-and-documentation">demos and documentation for your contribution</a>, or file an issue in the <a href="https://github.com/blackbaud/skyux2-docs">SKY UX docs repo</a> to work with a SKY UX writer. All components require demos to accompany the <a stacheRouterLink="/components">Components reference documentation</a>, and we recommend that you also create the reference documentation because it can be useful to uncover issues with your feature.
         </p>
+        <sky-restricted-view>
+          <sky-alert alertType="info">
+            Blackbaud developers should file documentation issues in ADO instead of GitHub. The SKY UX team uses ADO dashboards to manage work, but other than the location for filing issues, the process remains the same, including the availability of a <a href="https://blackbaud.visualstudio.com/Products/_workitems/create/Feature?templateId=553e8cbd-12ff-4e72-a44f-e47e2de5f0a8&ownerId=26bb0bf6-45ac-482d-b33e-02e252c297e3">template for filing issues</a>.
+          </sky-alert>
+        </sky-restricted-view>
+        <sky-restricted-content-alert></sky-restricted-content-alert>
```

Comment:
Same question on this page about 1 alert or two?

Reply:
As discussed, we'll go with 2 alerts here.